pkg/gomod/check.go

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gomod
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/blang/semver/v4"
+
+	"knative.dev/test-infra/pkg/git"
+)
+
+// Check examines a go mod file for dependencies and  determines if each have a release artifact
+// based on the ruleset provided.
+//
+// See Also
```

Comment:
see also what? :)

Reply:
it makes a godoc header: https://medium.com/@elliotchance/godoc-tips-tricks-cda6571549b

Reply:
Does it work with fucntion level headers or only on package level?

Reply:
in my ide, the preview looks correct with the header :idk:

Reply:
maybe not. I will drop it.